### PR TITLE
Openstack and VMware corrections

### PR DESCRIPTION
**Pull request by gellner**
Openstack FAQ
- Small change to ports required to include more important ports first.

Openstack GS
- First sentence was unnecessary.
- Changed some wording

VMware FAQ
- Removed reference to HA vShield Edges
- Removed out of date references to specific Linux versions
- Change language regarding up to date images/templates
- Added a couple of references to Journaling protection - this should probably be highlighted in preference to Geosyncronous.

### CODE REVIEW

File: articles/vmware/vmw-faq.md [hunk 1]
```diff
@@ -552,13 +541,15 @@ The recovery time objective (RTO) depends on the nature of the disaster or failu
 
 If you need more control and assurance around how data is replicated, the DR service is tested, or the solution handles failover and failback, we recommend you create your own DR solution by using independent sets of VMs in the various regions offered by the UKCloud platform.
 
+An alternative UKCloud solution to acheive DR with a low RPO and RTO is [Journaling Protection](vmw-sco-journaling-protection.md).
```
Comment (suehighmoor) [inline articles/vmware/vmw-faq.md]: typo: achieve

File: articles/vmware/vmw-faq.md [hunk 2]
```diff
@@ -376,7 +367,7 @@ UKCloud monitors the underlying platform but doesn't monitor customers' OSs or a
 
 ### Do you offer autoscaling?
 
-There's no standard product offering for autoscale, but the platform API can be used to do this with a little developer effort. UKCloud provides blueprints in the Knowledge Centre on the UKCloud Portal which offer guidance on this.
+There's no standard product offering for autoscale, but the platform API can be used to do this with a little developer effort. UKCloud provides blueprints in the Knowledge Centre which offer guidance on this.
```
Comment (suehighmoor) [inline articles/vmware/vmw-faq.md]: Blueprints aren't available on the Knowledge Centre at the moment, so can you change this to:
We provide blueprints on the UKCloud web site that offer guidance on this.

File: articles/vmware/vmw-faq.md [hunk 3]
```diff
@@ -552,13 +541,15 @@ The recovery time objective (RTO) depends on the nature of the disaster or failu
 
 If you need more control and assurance around how data is replicated, the DR service is tested, or the solution handles failover and failback, we recommend you create your own DR solution by using independent sets of VMs in the various regions offered by the UKCloud platform.
 
+An alternative UKCloud solution to acheive DR with a low RPO and RTO is [Journaling Protection](vmw-sco-journaling-protection.md).
+
 ### How does UKCloud enable customers to create their own DR solutions?
 
-Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or in the application.
+Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or into the application.
```
Comment (suehighmoor) [inline articles/vmware/vmw-faq.md]: Issue existed in the original text but while you're editing, could you change customers services to customers' services (add apostrophe)

File: articles/vmware/vmw-faq.md [hunk 4]
```diff
@@ -552,13 +541,15 @@ The recovery time objective (RTO) depends on the nature of the disaster or failu
...
 
 To help this process, the UKCloud platform has been designed to give you the options you need to design a disaster tolerant solution. UKCloud's platform is about providing customers with choice. Clearly, customers need to balance the low probability of these failure scenarios occurring with the cost and complexity involved in mitigating their impact.
 
-Depending on the type of failure you are trying to mitigate, UKCloud has been developed to allow you design appropriate resilience into your solution.
+Depending on the type of failure you are trying to mitigate, UKCloud services have been developed to allow you design appropriate resilience into your solution.
```
Comment (suehighmoor) [inline articles/vmware/vmw-faq.md]: Issue existed in the original text but while you're editing, could you change
developed to allow you design
to
developed to enable you to design

(change allow to enable and add the missing to)

File: articles/openstack/ostack-gs.md [hunk 5]
```diff
@@ -64,7 +58,8 @@ You can create your network infrastructure in one of the following ways:
 
 - Using Horizon to manually create your infrastructure (see [Using Horizon to manually create your infrastructure](#using-horizon-to-manually-create-your-infrastructure))
 - Using the OpenStack Heat orchestration tool to create stacks (see [Using the OpenStack Heat orchestration tool to create stacks](#using-the-openstack-heat-orchestration-tool-to-create-stacks))
-- Using the OpenStack API and automation (see [Using the OpenStack API and automation](#using the-openstack-api-and-automation))
+- Using the OpenStack API and automation
```
Comment (suehighmoor) [inline articles/openstack/ostack-gs.md]: Could you put the link back in:

(see [Using the OpenStack API and automation](#using-the-openstack-api-and-automation)

(in the original text it was missing a hypen, which is why the link was broken)

**Comment by suehighmoor** [comment]
@gellner changes have been approved by two of your reviewers (one technical, one doc) so we can go ahead and publish now unless you require feedback from any of the other reviewers you listed.

**Comment by gellner** [comment]
Thanks Sue, Steve. I'm happy for this to be Merged. I don't think the VMware changes are controversial (those will follow soon in an email discussion...), and S Relf has approved from CNI team.

**Pull request merged**